Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride.hs

```diff
@@ -119,13 +121,13 @@ listDriverRides ::
   Maybe DRide.RideStatus ->
   m DriverRideListRes
 listDriverRides driverId mbLimit mbOffset mbOnlyActive mbRideStatus = do
-  rides <- runInReplica $ QRide.findAllByDriverId driverId mbLimit mbOffset mbOnlyActive mbRideStatus
+  rides <- QRide.findAllByDriverId driverId mbLimit mbOffset mbOnlyActive mbRideStatus
   driverRideLis <- forM rides $ \(ride, booking) -> do
     rideDetail <- runInReplica $ QRD.findById ride.id >>= fromMaybeM (VehicleNotFound driverId.getId)
     rideRating <- runInReplica $ QR.findRatingForRide ride.id
     driverNumber <- RD.getDriverNumber rideDetail
     mbExophone <- CQExophone.findByPrimaryPhone booking.primaryExophone
-    pure $ mkDriverRideRes rideDetail driverNumber rideRating mbExophone (ride, booking)
+    pure $ mkDriverRideRes rideDetail driverNumber rideRating mbExophone (ride, booking) ride.fromLocation ride.toLocation
```

Comment:
ride already includes toLocation and fromLocation. No need to pass them separately